src/foam/java/refinements.js

```diff
@@ -861,11 +922,16 @@ foam.CLASS({
     ['javaJSONParser', 'foam.lib.json.BooleanParser'],
     ['javaInfoType', 'foam.core.AbstractBooleanPropertyInfo']
   ],
+
   methods: [
     function createJavaPropertyInfo_(cls) {
       var info = this.SUPER(cls);
+      var c = info.getMethod('comparePropertyValue');
+      c.body = 'return compareValues((boolean) key, (boolean) f(o));';
```

Comment:
Isn't this just a duplicate of the implementation already provided by AbstractBooleanPropertyInfo?